content/realtime/channels.textile

```diff
@@ -645,6 +691,17 @@ bc[swift]. channel.on(.attached) { stateChange in
   }
 }
 
+bc[flutter]. final stateChangeListener = channel.on(ably.ChannelEvent.attached)
+  .listen((ably.ChannelStateChange state){
+    print('channel ${channel.name} is now attached');
+    if (stateChange.resumed) {
+    System.out.println('Message continuity was preserved');
```

Comment:
Inconsistent indentation.

Reply:
addressed in e288898cb0f972b74350bd69c2da054346fbb4f0